privacy-config/privacy-config-impl/src/main/java/com/duckduckgo/privacy/config/impl/features/trackingparameters/RealTrackingParameters.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.privacy.config.impl.features.trackingparameters
+
+import android.net.Uri
+import androidx.core.net.toUri
+import com.duckduckgo.app.global.UriString
+import com.duckduckgo.app.global.domain
+import com.duckduckgo.app.global.replaceQueryParameters
+import com.duckduckgo.app.userwhitelist.api.UserWhiteListRepository
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.FeatureToggle
+import com.duckduckgo.privacy.config.api.PrivacyFeatureName
+import com.duckduckgo.privacy.config.api.TrackingParameters
+import com.duckduckgo.privacy.config.impl.features.unprotectedtemporary.UnprotectedTemporary
+import com.duckduckgo.privacy.config.store.features.trackingparameters.TrackingParametersRepository
+import com.squareup.anvil.annotations.ContributesBinding
+import timber.log.Timber
+import java.lang.UnsupportedOperationException
+import javax.inject.Inject
+
+@ContributesBinding(AppScope::class)
+class RealTrackingParameters @Inject constructor(
```

Comment:
I'm actually going to remove the "Detector" from that feature, it's going to be renamed "AmpLinks", since that is the name of the feature. That will be inline with the naming convention of the other features.